src/lib/plugin_apis/dm.api

```diff
@@ -88,6 +88,15 @@ gchar* bd_dm_name_from_node (const gchar *dm_node, GError **error);
 gchar* bd_dm_node_from_name (const gchar *map_name, GError **error);
 
 /**
+ * bd_dm_get_subsystem_from_name:
+ * @device_name: name of the device
+ * @error: (out): place to store error (if any)
+ *
+ * Returns: subsystem of the given device
```

Comment:
Please also specify ``Tech category: %BD_DM_TECH_RAID-%BD_DM_TECH_QUERY`` here (and in the *.c* file).

Reply:
Fixed.